lib/tasks/etl.rake

```diff
@@ -15,6 +15,19 @@ namespace :etl do
     end
   end
 
+  desc 'Delete existing metrics and Run Etl Master process across a range of dates'
+  task :repopulate_master, %i[from to] => [:environment] do |_t, args|
```

Comment:
Can we rename this task to `rerun_master` or something similar?

Reply:
Sure, give me a minute